api/control.yaml

```diff
@@ -256,6 +344,181 @@ components:
         primary:
           description: If the email is primary.
           type: boolean
+    GrantType:
+      description: |
+        Grant type identifier, from RFC 8693 Section 2.
+        See https://datatracker.ietf.org/doc/html/rfc8693#section-2
+      type: string
+      enum:
+        - urn:ietf:params:oauth:grant-type:token-exchange
+      # NOTE:
+      # OpenAPI 3.0/2.0 do not have a way to define custom names for enum values,
+      # but some tools (e.g., openapi-generator and openapi-typescript) provide
+      # x-prefixed extensions for this purpose.
+      x-enum-varnames:
+        - TOKEN_EXCHANGE
+    TokenType:
+      description: |
+        Token type identifier, from RFC 8693 Section 3.
+        See https://datatracker.ietf.org/doc/html/rfc8693#section-3
+      type: string
+      enum:
+        - urn:ietf:params:oauth:token-type:access_token
+        - urn:ietf:params:oauth:token-type:refresh_token
+        - urn:ietf:params:oauth:token-type:id_token
+        - urn:ietf:params:oauth:token-type:saml1
+        - urn:ietf:params:oauth:token-type:saml2
+        - urn:ietf:params:oauth:token-type:jwt
+      # NOTE:
+      # OpenAPI 3.0/2.0 do not have a way to define custom names for enum values,
+      # but some tools (e.g., openapi-generator and openapi-typescript) provide
+      # x-prefixed extensions for this purpose.
+      x-enum-varnames:
+        - ACCESS_TOKEN
+        - REFRESH_TOKEN
+        - ID_TOKEN
+        - SAML1
+        - SAML2
+        - JWT
+    AccessTokenType:
+      description: |
+        Access token type identifier, from RFC 8693 Section 2.
+        See https://datatracker.ietf.org/doc/html/rfc8693#section-2
+      type: string
+      enum:
+        - Bearer
+        - MAC
+        - N_A
+    ResponseType:
+      description: |
+        Authorization endpoint response type, from RFC 6749 Section 3.
+        See https://datatracker.ietf.org/doc/html/rfc6749#section-3
+      type: string
+      enum:
+        - code
+        - token
+    AuthorizationGrantType:
+      description: |
+        Authorization grant type, from RFC 6749 Section 4.
+        See https://datatracker.ietf.org/doc/html/rfc6749#section-4
+      type: string
+      enum:
+        - authorization_code
```

Comment:
similar, do we want to include all posible grant types, or just what we support? feels a bit misleading to include types that are not usable.